nilearn/maskers/surface_labels_masker.py

```diff
@@ -572,28 +572,29 @@ def _create_figure_for_report(self):
             img = mean_img(img)
             vmin, vmax = img.data._get_min_max()
 
-        # TODO: possibly allow to generate a report with other views
-        views = ["lateral", "medial"]
-        hemispheres = ["left", "right"]
+        hem_view = {
+            "left": ["lateral", "medial"],
+            "right": ["lateral", "medial"],
+            "both": ["anterior", "posterior"],
+        }
```

Comment:
Not sure I understand your aim here: you want to plot left, right AND both ?

Reply:
not related to this PR, but if we could find a way to have a single coloar bar for all plots, it'd be nice